angular/package.json

```diff
@@ -133,6 +133,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "~1.19.1",
     "puppeteer": "^2.1.1",
+    "sass": "^1.53.0",
```

Comment:
The `package-lock.json` has this:
```
"node-sass": "4.13.1"
```
Not too familiar with frontend, but given that Dart-Sass (`sass`) is the successor to `node-sass`, wouldn't there be a potential to break the CSS, given that it's essentially bumping up the version number?